wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.0.0-beta8.0-RC5.0/1__change_preferences_type.sql

```diff
@@ -0,0 +1,17 @@
+--
+-- Copyright (c) 2012-2019 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+-- component preferences
+
+ALTER TABLE devfile_component_preferences MODIFY COLUMN preference TEXT;
```

Comment:
Looks like migration script file should be renamed.

Reply:
you mean wrong version? yes.

Reply:
yes